php/Terminus/Site.php

```diff
@@ -292,25 +292,54 @@ public function import($url, $element) {
   }
 
   /**
-   * get site instrument
+   * Changes payment instrument of given site
+   *
+   * @params [string] $uuid UUID of new payment instrument
+   * @return [Workflow] $workflow Workflow object for the request
    */
-  public function instrument($org=null) {
-    $path = 'instrument';
-    $method = 'GET';
-    $options = null;
-    if ($org) {
-      $path = 'workflows';
-      $data = array(
-          'type'    => 'transfer_site_payment_to_organization',
-          'params'  => array(
-              'to_organization_id' => $org,
-          ),
-        );
-      $options = array( 'body' => json_encode($data) , 'headers'=>array('Content-type'=>'application/json') );
-      $method = 'POST';
+  public function changeInstrument($uuid = 0) {
```

Comment:
I really like polymorphic interfaces. It's a core functionality of PHP. Without it, we'll end up writing a whole lot more code than we other wise would to say the same things.